postgresql_blueprints/upload_file.py

```diff
@@ -128,58 +128,49 @@ def combine_folder_and_file_name(folder_name, file_name):
     return combined_name
 
 
+def determine_upload_method(db_connection):
+    if 'db.bit.io' in str(db_connection):
+        upload_method = bitio_upload_method
```

Comment:
bitio_upload_method is a function? Why are you setting that to a variable?

Reply:
To be honest, I'm not sure on the specifics. That method was directly pulled from bit.io documentation and they call it without the `()`.

https://docs.bit.io/docs/connecting-via-pandas#loading-dataframes-quickly-to-bitio